diff --git a/build/Linux+BSD/musescore.xml.in b/build/Linux+BSD/musescore.xml.in
index 107f80f643..fc41b46d2d 100644
--- a/build/Linux+BSD/musescore.xml.in
+++ b/build/Linux+BSD/musescore.xml.in
@@ -7,7 +7,7 @@
         <comment>MuseScore File</comment>
         <glob pattern="*.mscz"/>
         <sub-class-of type="application/zip"/>
-        <icon name="application-x-musescore@MUSESCORE_INSTALL_SUFFIX@"/>
+        <icon name="org.musescore.MuseScore.application-x-musescore@MUSESCORE_INSTALL_SUFFIX@"/>
     </mime-type>
 
     <mime-type type="application/x-musescore+xml">
@@ -15,7 +15,7 @@
         <glob pattern="*.mscx"/>
         <sub-class-of type="application/xml"/>
         <root-XML namespaceURI="" localName="museScore"/>
-        <icon name="application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
+        <icon name="org.musescore.MuseScore.application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
         <magic>
             <match type="string" value="&lt;?xml" offset="0">
                 <match type="string" value="&lt;museScore" offset="0:128">
@@ -30,7 +30,7 @@
         <glob pattern="*.mscsx"/>
         <sub-class-of type="application/xml"/>
         <root-XML namespaceURI="" localName="museScore"/>
-        <icon name="application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
+        <icon name="org.musescore.MuseScore.application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
         <magic>
             <match type="string" value="&lt;?xml" offset="0">
                 <match type="string" value="&lt;museScore" offset="0:128">
@@ -44,7 +44,7 @@
         <comment>MuseScore compressed backup score</comment>
         <glob pattern="*.mscz,"/>
         <sub-class-of type="application/zip"/>
-        <icon name="application-x-musescore@MUSESCORE_INSTALL_SUFFIX@"/>
+        <icon name="org.musescore.MuseScore.application-x-musescore@MUSESCORE_INSTALL_SUFFIX@"/>
     </mime-type>
 
     <mime-type type="application/x-musescore+xml">
@@ -52,7 +52,7 @@
         <glob pattern="*.mscx,"/>
         <sub-class-of type="application/xml"/>
         <root-XML namespaceURI="" localName="museScore"/>
-        <icon name="application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
+        <icon name="org.musescore.MuseScore.application-x-musescore@MUSESCORE_INSTALL_SUFFIX@+xml"/>
         <magic>
             <match type="string" value="&lt;?xml" offset="0">
                 <match type="string" value="&lt;museScore" offset="0:128">
@@ -105,18 +105,17 @@
         <comment>MusicXML compressed score</comment>
         <glob pattern="*.mxl"/>
         <sub-class-of type="application/zip"/>
-        <icon name="application-vnd.recordare.musicxml@MUSESCORE_INSTALL_SUFFIX@"/>
+        <icon name="org.musescore.MuseScore.application-vnd.recordare.musicxml@MUSESCORE_INSTALL_SUFFIX@"/>
     </mime-type>
 
     <mime-type type="application/vnd.recordare.musicxml+xml">
         <!-- http://www.musicxml.com/for-developers/musicxml-dtd/ -->
         <comment>MusicXML uncompressed score</comment>
         <glob pattern="*.musicxml"/><!-- preferred extension must go first -->
-        <glob pattern="*.xml" weight="40"/><!-- reduced weight (default 50) to avoid matching ordinary XML files -->
         <sub-class-of type="application/xml"/>
         <root-XML namespaceURI="" localName="score-partwise"/>
         <root-XML namespaceURI="" localName="score-timewise"/>
-        <icon name="application-vnd.recordare.musicxml@MUSESCORE_INSTALL_SUFFIX@+xml"/>
+        <icon name="org.musescore.MuseScore.application-vnd.recordare.musicxml@MUSESCORE_INSTALL_SUFFIX@+xml"/>
         <magic>
             <match type="string" value="&lt;?xml" offset="0">
                 <match type="string" value="score-partwise" offset="0:128"/>
