diff --git a/build/Linux+BSD/musescore.xml.in b/build/Linux+BSD/musescore.xml.in
index 9de194104..e7b706511 100644
--- a/build/Linux+BSD/musescore.xml.in
+++ b/build/Linux+BSD/musescore.xml.in
@@ -3,17 +3,20 @@
   <mime-type type="application/x-musescore@MSCORE_INSTALL_SUFFIX@">@Variables_substituted_by_CMAKE_on_installation@
     <comment>MuseScore file</comment>
     <sub-class-of type="application/zip"/>
+    <icon name="org.musescore.MuseScore.application-x-musescore@MSCORE_INSTALL_SUFFIX@"/>
     <glob pattern="*.mscz"/>
   </mime-type>
   <mime-type type="application/x-musescore@MSCORE_INSTALL_SUFFIX@+xml">
     <comment>uncompressed MuseScore file</comment>
     <sub-class-of type="application/xml"/>
+    <icon name="org.musescore.MuseScore.application-x-musescore@MSCORE_INSTALL_SUFFIX@+xml"/>
     <glob pattern="*.mscx" />
   </mime-type>
   <mime-type type="application/vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@">
     <!-- http://www.musicxml.com/for-developers/musicxml-dtd/ -->
     <_comment>compressed MusicXML file</_comment>
     <sub-class-of type="application/zip"/>
+    <icon name="org.musescore.MuseScore.application-vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@"/>
 <!-- Note: a custom icon is used for MusicXML files. You can change this below -->
 <!--  <icon name="application-x-musescore"/> Uncomment to use MuseScore file icon -->
 <!--  <generic-icon name="audio-x-generic"/> Uncomment to use generic audio file icon -->
@@ -23,6 +26,7 @@
     <!-- http://www.musicxml.com/for-developers/musicxml-dtd/ -->
     <_comment>uncompressed MusicXML file</_comment>
     <sub-class-of type="application/xml"/>
+    <icon name="org.musescore.MuseScore.application-vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@+xml"/>
 <!--  <icon name="application-x-musescore"/> Uncomment to use MuseScore file icon -->
 <!--  <generic-icon name="audio-x-generic"/> Uncomment to use generic audio file icon -->
     <magic>
@@ -41,6 +45,7 @@
         <!-- http://www.musicxml.com/for-developers/musicxml-dtd/ -->
         <_comment>uncompressed MusicXML file</_comment>
         <sub-class-of type="application/xml"/>
+        <icon name="org.musescore.MuseScore.application-vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@+xml"/>
         <!--  <icon name="application-x-musescore"/> Uncomment to use MuseScore file icon -->
         <!--  <generic-icon name="audio-x-generic"/> Uncomment to use generic audio file icon -->
         <magic>
