diff --git a/src/framework/vst/CMakeLists.txt b/src/framework/vst/CMakeLists.txt
index 196fff93b7..15c5834534 100644
--- a/src/framework/vst/CMakeLists.txt
+++ b/src/framework/vst/CMakeLists.txt
@@ -104,13 +104,6 @@ set(MODULE_LINK
     vst_sdk_3
     )
 
-if (OS_IS_LIN)
-    set(MODULE_LINK
-        ${MODULE_LINK}
-        stdc++fs
-        )
-endif()
-
 if (QT_SUPPORT)
     list(APPEND MODULE_LINK Qt::Qml Qt::Widgets)
 endif()
