diff --git a/build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in b/build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in
index d52ce502f..8945cc8f2 100644
--- a/build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in
+++ b/build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in
@@ -82,6 +82,56 @@
     <content_attribute id="money-purchasing">mild</content_attribute>
   </content_rating>
   <releases>
+    <release date="2020-01-25" version="3.4.1">
+      <url>https://musescore.org/en/3.4.1</url>
+      <description>
+        <p>Fixes:</p>
+        <ul>
+          <li>MuseScore crashed after closing a menu bar pop-up window if no score is opened</li>
+          <li>Audio glitches on note input and playback happened on macOS and other platforms</li>
+          <li>Parts corruption happened on timewise delete of individual beats</li>
+          <li>Crash happened when undoing "Beam middle" setting on a single note</li>
+          <li>Pedal lines alignment applied to the whole system, not individual staff</li>
+        </ul>
+      </description>
+    </release>
+    <release date="2020-01-23" version="3.4">
+      <url>https://musescore.org/en/3.4</url>
+      <description>
+        <p>New:</p>
+        <ul>
+          <li>Apply palette elements with a single click if there is a selection in the score</li>
+          <li>Allow dragging notes horizontally</li>
+          <li>Slurs, hairpins and other elements can be edited after a single click</li>
+          <li>Add middle adjustment handle for beams, for moving whole beam horizontally</li>
+          <li>Add command-line option for transposing scores</li>
+        </ul>
+        <p>Improvements:</p>
+        <ul>
+          <li>Double click is not needed to reach edit mode anymore</li>
+          <li>Introduce bend properties in the Inspector</li>
+          <li>Allow access to several properties window in the Inspector</li>
+          <li>Name of the newly created custom palette can be specified</li>
+          <li>Accessibility: improve speech for elements with spanners</li>
+          <li>Embed Tremolo Bar editor to the inspector</li>
+        </ul>
+        <p>Fixes:</p>
+        <ul>
+          <li>"Don't play trill" option silenced the note playback</li>
+          <li>Slurs on small staves were displaced in some cases</li>
+          <li>Barline handles were drawn incorrectly after dragging one</li>
+          <li>Strings in the Part dialogue were ambiguous</li>
+          <li>Y Offset value of fretboards didn't restore after undoing the values being changed from Edit Mode</li>
+          <li>Replacing a note with an accidental left the accidental on the new note</li>
+          <li>Adding Intervals (above/below) didn't take into consideration the accidental toggle state</li>
+          <li>Multiple chord symbols attached to same note didn't copy as part of the range</li>
+          <li>Strings in fret diagrams without "X" or "O" displayed as "?" on Linux</li>
+          <li>MuseScore crashed when changing a triplet's rest's duration</li>
+          <li>Images attached to rests weren't imported from MuseScore 2</li>
+          <li>Tremolo Bar dialog had multiple UX issues</li>
+        </ul>
+      </description>
+    </release>
     <release date="2019-12-04" version="3.3.4">
        <url>https://musescore.org/en/3.3.4</url>
        <description>
