diff --git a/public.sdk/source/vst/hosting/module_linux.cpp b/public.sdk/source/vst/hosting/module_linux.cpp
index 404054c..d9a2938 100644
--- a/public.sdk/source/vst/hosting/module_linux.cpp
+++ b/public.sdk/source/vst/hosting/module_linux.cpp
@@ -286,7 +286,7 @@ Module::PathList Module::getModulePaths ()
 	 * Application				: /$APPFOLDER/vst3/
 	 */
 
-	const auto systemPaths = {"/usr/lib/vst3/", "/usr/local/lib/vst3/"};
+	const auto systemPaths = {"/app/lib/vst3/", "/app/extensions/Plugins/vst3/"};
 
 	PathList list;
 	if (auto homeDir = getenv ("HOME"))
